lib/tasks/audit_code.rake

```diff
@@ -372,9 +383,23 @@ def print_repo_file_diffs(repolatest, repo, fname, user_name, safe_revision)
     puts 'Unknown repo'
   end
 
-  puts %(To flag the changes to this file as safe, run:)
-  puts "  #{rake_cmd} audit:safe release=#{repolatest} file=#{fname} reviewed_by=#{user_name}" \
-       ' comments=""'
+  if interactive
+    response = ask("Flag #{fname} changes safe? [Yes|No|Abort]: ") { |q| q.in = %w[Yes No Abort] }
```

Comment:
Justa little thing - another option here would be to use highline's `agree` method, which will handles variants of `yes/no/y/n/Y/N` etc. Wouldn't support the third "abort" option though.

...any way we can make this case-insensitive / partially-matching?